app/api/video_recordings.py

```diff
@@ -19,39 +19,48 @@ def before_get(self, args, kwargs):
         if kwargs.get('video_stream_id'):
             stream = safe_query_kwargs(VideoStream, kwargs, 'video_stream_id', 'id')
 
-            if not has_access('is_organizer', event_id=stream.event_id):
-                raise ForbiddenError(
-                    {'pointer': 'event_id'},
-                    'You need to be the event organizer to access video recordings.',
-                )
+            if stream.channel and stream.channel.provider == 'bbb':
+                if not has_access('is_organizer', event_id=stream.event_id):
+                    raise ForbiddenError(
+                        {'pointer': 'event_id'},
+                        'You need to be the event organizer to access video recordings.',
+                    )
```

Comment:
Everything works! I think we should keep these lines above the if statement as they are for permissions.

Reply:
@pc-beast Why to check for permission when the stream is not of bbb

Reply:
This function will run everytime the api is called. Regardless of permissions. We can restrict the further execution ang give correct message to user by using permission above.